sim/shaman/lightning_shield.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/wowsims/wotlk/sim/core/proto"
 )
 
+const defaultLightningSheildStacks = 3
+const staticShockStackMultiplier = 2
```

Comment:
Looked into this. Seems it's following convention by "private non-exported constant values" to have `lowerCase`.

Reply:
just trying to give some names to some magic numbers.